Common/Source/Devices/devLXNano3.cpp

```diff
@@ -2262,7 +2264,7 @@ TCHAR szName[MAX_VAL_STR_LEN];
       szName, DegLat, MinLat, NoS, DegLon, MinLon, EoW,
       (int) (WayPointList[overindex].Altitude +0.5));
 
-      _tcsncat (szName, _T(" ($PLXVTARG)"),MAX_VAL_STR_LEN);
+      _tcsncat (szName, _T(" ($PLXVTARG)"),MAX_NMEA_LEN);
```

Comment:
this a strncat common  misuse !
https://help.semmle.com/wiki/display/CCPPOBJ/Potentially+unsafe+call+to+strncat